service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java

```diff
@@ -10,218 +10,169 @@
      */
     public void setETag(String eTag) throws IllegalArgumentException
     {
         if (Tools.isNullOrEmpty(eTag))
         {
-            /*
-            **Codes_SRS_DEVICETWINDEVICE_21_029: [** The seteTag shall throw IllegalArgumentException if the input string is empty or null.**]**
-             */
-            throw new IllegalArgumentException("ETag cannot be null or empty");
+            throw new IllegalArgumentException("ETag cannot be null or empty.");
         }
 
-        /*
-        **Codes_SRS_DEVICETWINDEVICE_21_030: [** The seteTag shall store the eTag.**]**
-         */
         this.eTag = eTag;
     }
 
     /**
-     * Getter for the eTag
+     * Getter for the ETag.
      *
-     * @return the stored eTag. It will be {@code null} if not set.
+     * @return The stored ETag. It will be {@code null} if not set.
      */
     public String getETag()
     {
-        /*
-        **Codes_SRS_DEVICETWINDEVICE_21_031: [** The geteTag shall return the stored eTag.**]**
-         */
         return this.eTag;
     }
 
     /**
-     * Setter for Twin version
+     * Setter for the twin version.
      *
-     * @param version is the value of the version
+     * @param version The version of the twin.
      */
     void setVersion(Integer version)
     {
-        /*
-        **Codes_SRS_DEVICETWINDEVICE_21_032: [** The setVersion shall store the Twin version.**]**
-         */
         this.version = version;
     }
 
     /**
-     * Getter for the Twin version
+     * Getter for the twin version.
      *
-     * @return the stored version. It can be {@code null}.
+     * @return The stored version. It can be {@code null}.
      */
     public Integer getVersion()
     {
-        /*
-        **Codes_SRS_DEVICETWINDEVICE_21_033: [** The getVersion shall return the stored Twin version.**]**
-         */
         return this.version;
     }
 
     /**
-     * Setter for the tags
+     * Setter for tags.
      *
-     * @param tags A set of tag key-value pairs
-     * @throws IllegalArgumentException This exception is thrown if the set tags is null
+     * @param tags A set of tag key/value pairs.
+     * @throws IllegalArgumentException This exception is thrown if the set tags is {@code null}.
      */
     public void setTags(Set<Pair> tags) throws IllegalArgumentException
     {
         if (tags == null)
         {
-            /*
-            **Codes_SRS_DEVICETWINDEVICE_25_008: [** If the tags Set is null then this method shall throw IllegalArgumentException.**]**
-             */
             throw new IllegalArgumentException("tags cannot be null");
         }
-        /*
-        **Codes_SRS_DEVICETWINDEVICE_25_007: [** This method shall convert the set of pairs of tags to a map and save it. **]**
-         */
         this.tag = this.setToMap(tags);
     }
 
     /**
-     * Getter to get Tags Set
-     * @return A set of tag key value pairs.
+     * Getter to get the tags setter.
```

Comment:
I'm not sure we could word it more confusingly. Ideas?

Reply:
"Getter for the twin tags"?

Reply:
but it is a getter to get the setter